Modules/Vector/Vector3D.py

```diff
@@ -0,0 +1,13 @@
+#/usr/bin/python3
+from math import sqrt
+class vector3D:
+    def __init__(self,x:float=0,y:float=0,z:float=0):
+        self.x = x
+        self.y = y
+        self.z = z
+    def distanceTo(self,vector)->int:
```

Comment:
can you rename this to `distance_to`? Or just `distance` if you don't like the underscore.